src/pages/ConnectWallet/components/WalletList.tsx

```diff
@@ -0,0 +1,163 @@
+import { Alert, AlertDescription, AlertIcon, Center, Spinner, Stack } from '@chakra-ui/react'
+import { useQuery } from '@tanstack/react-query'
+import { useCallback, useMemo, useState } from 'react'
+import { useHistory } from 'react-router'
+import { MobileWalletDialogRoutes } from 'components/MobileWalletDialog/types'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useLocalWallet } from 'context/WalletProvider/local-wallet'
+import { MobileConfig } from 'context/WalletProvider/MobileWallet/config'
+import { getWallet, listWallets } from 'context/WalletProvider/MobileWallet/mobileMessageHandlers'
+import type { RevocableWallet } from 'context/WalletProvider/MobileWallet/RevocableWallet'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { WalletCard } from './WalletCard'
+
+export type WalletInfo = {
+  id?: string
+  name?: string
+  createdAt?: number
```

Comment:
Are those actually optional?

Reply:
Actually doesn't seem like they are - can we reuse the `MobileWalletInfo` type from `src/context/WalletProvider/MobileWallet/types.ts`? 
We now have 4 (previously 3) duplicate `WalletInfo` types